public/app/exercise/exercise.controller.js

```diff
@@ -793,6 +804,7 @@
           break;
         }
       }
+      loadEditorContent();
```

Comment:
`setIDEMode()` should not update the value of the editor, it is not its purpose.
It would be better to let [`updateUI()`](https://github.com/Narxem/webPLM/blob/master/public/app/exercise/exercise.controller.js#L843-L844)  update it if needed.